.zuul.yaml

```
@@ -87,6 +97,7 @@
       STF CRC jobs that build and deploy STF
     github-check:
       jobs:
+        - pvs_crc
         - stf-crc-ocp_412-nightly_bundles
         - stf-crc-ocp_412-local_build
         - stf-crc-ocp_413-nightly_bundles
```

These jobs take quite a while to run.

For testing, I recommend that you comment them out so that you can get results faster from the ``pvs_crc`` job.

```suggestion
        #- stf-crc-ocp_412-nightly_bundles
        #- stf-crc-ocp_412-local_build
        #- stf-crc-ocp_413-nightly_bundles
```